build_runner/lib/src/entrypoint/options.dart

```diff
@@ -374,7 +374,7 @@ class _ServeCommand extends _WatchCommand {
           'args in <dir>[:<port>] format.');
     } else {
       for (var target in options.serveTargets) {
-        stdout.writeln('Serving `${target.dir}` on port ${target.port}');
+        stdout.writeln('Serving `${target.dir}` on http://${options.hostName}:${target.port}');
```

Comment:
This will be slightly odd for the magic hostName `any`... but I think that's probably OK.